shared-test/uk/gov/hmrc/thirdpartyapplication/ApplicationStateUtil.scala

```diff
@@ -24,6 +24,11 @@ trait ApplicationStateUtil {
 
   def testingState() = ApplicationState(name = State.TESTING)
 
+  def preProductionState(requestedBy: String) = ApplicationState(
+    name = State.PRE_PRODUCTION,
+    requestedByEmailAddress = Some(requestedBy),
+    verificationCode = Some(generatedVerificationCode))
```

Comment:
and again.  We need to check what this is meant to be.  I think None would work here.